### ISSUE TITLE
Suggestion - ModPE.importScript(par1String);

### ISSUE THREAD

**Issue by ghost** [MCMrARM/toolbox-issue-tracker#24]
Suggestion:
ModPE.importScript(par1String);

**What should it do?**
Open a toolbox screen with the text "The script "SomeScript" wants to import the script "another script""
And two buttons (Import and Cancel)

**Comment by slice** [comment]
It would also have to take a callback because dialog boxes run on a separate thread.

**Comment by ghost** [comment]
It would be awesome if you could import from download websites (like MediaFire). Then i could make a very easy AutoUpdater

**Comment by ghost** [comment]
Yeah there could be two options, import from File and import from website (Github or pastebin)

**Comment by Johnmacrocraft** [comment]
@Godsoft029 What if it was sightly able to be security problem
